source/core/slang-uint-set.h

```diff
@@ -149,6 +149,13 @@ class UIntSet
         Index block = 0;
         Element processedElement = 0;
         Element bitUnset = 0;
+
+        void processBitUnset()
```

Comment:
I don't quite understand why we need to hold `bitUnset` here. It seems to be holding a value where only 1 bit is set, which is the next bit we are iterating. Can't we simply achieve this bit by calling `bitscanForward` on `processedElement`?

Also the function can be renamed to `clearLeastSignificantBit` to be more descriptive.

Reply:
> Can't we simply achieve this bit by calling bitscanForward on processedElement?

This does work (and is probably better todo)